packages/pigeon/CHANGELOG.md

```diff
@@ -1,285 +1,289 @@
+## 3.2.10
+
+- Expose SwiftOptions
+
 ## 3.2.9
 
-* Updates text theme parameters to avoid deprecation issues.
+- Updates text theme parameters to avoid deprecation issues.
```

Comment:
Please don't make changes that violate our style guide. We use `*`.

Reply:
Oops, autoformatting of VSCode a bit aggressive here; fixed